modules/integration/tests-common/integration-tests-utils/src/main/java/org/wso2/mb/integration/common/utils/ui/pages/main/HomePage.java

```diff
@@ -34,39 +34,104 @@
 
     /**
      * Checks whether the current page is the home page. if not throws a runtime exception
+     *
      * @param driver WebDriver
      */
     public HomePage(WebDriver driver) {
         super(driver);
         // Check that we're on the right page.
         if (!driver.findElement(By.id(UIElementMapper.getInstance()
-                .getElement("home.dashboard.middle.text"))).getText().contains("Home")) {
+                                              .getElement("home.dashboard.middle.text"))).getText()
+                .contains("Home")) {
```

Comment:
Fixed in https://github.com/hemikak/product-mb/commit/0c5b0a8a444a2c8dabf347f40e34698c0a980017